assets/prototype/domain/shared/predicates/tickets/filters/sortTicketsList.ts

```diff
@@ -1,23 +1,24 @@
 /**
  * External dependencies
  */
-import { ascend, prop, sortWith } from 'ramda';
+import { ascend, prop, propOr, sortWith } from 'ramda';
 
 /**
  * Internal dependencies
  */
-import { DatesSortedBy } from '../../../../eventEditor/data/date/types';
+import { TicketsSortedBy } from '../../../../eventEditor/data/ticket/types';
+import { Ticket } from '../../../../eventEditor/data/types';
 
-const sortTicketsList = (tickets, order = 'chronologically') => {
+const sortTicketsList = (tickets: Ticket[], order = TicketsSortedBy.chronologically) => {
 	switch (order) {
-		case DatesSortedBy.chronologically:
-			return sortWith([ascend(prop('name')), ascend(prop('id'))], tickets);
-		case DatesSortedBy.byName:
-			return sortWith([ascend(prop('name'))], tickets);
-		case DatesSortedBy.byId:
+		case TicketsSortedBy.chronologically:
+			return sortWith([ascend(propOr(null, 'name')), ascend(prop('id'))], tickets);
+		case TicketsSortedBy.byName:
+			return sortWith([ascend(propOr(null, 'name'))], tickets);
+		case TicketsSortedBy.byId:
```

Comment:
`TicketsSortedBy.chronologically`, `TicketsSortedBy.byName`, `TicketsSortedBy.byId` all look a little weird because of the duplicated use of "by"

What about following the same thing you did for the `ShowTickets` type and rename `TicketsSortedBy` to `SortTickets` which would then read much better: `SortTickets.chronologically`, `SortTickets.byName`, `SortTickets.byId`

Reply:
Poetry level suggestion, thx :bowing_man: 
https://github.com/eventespresso/event-espresso-core/pull/2150/commits/b8bde44757816a7d04453c466b4253184d74ce7c